install/kubernetes/cilium/templates/cilium-configmap.yaml

```
@@ -443,6 +443,11 @@ data:
   # policy map (per endpoint)
   bpf-policy-map-max: "{{ .Values.bpf.policyMapMax | int }}"
 {{- end }}
+{{- if hasKey .Values.bpf "policyStatsMapMax" }}
+  # bpf-policy-stats-map-max specifies the maximum number of entries in global
+  # policy stats map
+  bpf-policy-stats-map-max: "{{ .Values.bpf.policyStatsMapMax }}"
```

```suggestion
  bpf-policy-stats-map-max: "{{ .Values.bpf.policyStatsMapMax || int }}"
```

See https://github.com/cilium/cilium/issues/34010 for details on why

---

Fixed, thanks!